packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderFilterIcon.tsx

```diff
@@ -0,0 +1,69 @@
+import { IconButton } from '@material-ui/core';
+import Badge from '@material-ui/core/Badge';
+import Tooltip from '@material-ui/core/Tooltip';
+import * as React from 'react';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { preferencePanelStateSelector } from '../../hooks/features/preferencesPanel/preferencePanelSelector';
+import { PreferencePanelsValue } from '../../hooks/features/preferencesPanel/preferencesPanelValue';
+import { useIcons } from '../../hooks/utils/useIcons';
+import { optionsSelector } from '../../hooks/utils/useOptionsProp';
+import { ApiContext } from '../api-context';
+
+export interface ColumnHeaderFilterIconProps {
+  counter?: number;
+}
+
+export const ColumnHeaderFilterIcon: React.FC<ColumnHeaderFilterIconProps> = (props) => {
+  const { counter } = props;
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const preferencePanel = useGridSelector(apiRef, preferencePanelStateSelector);
+
+  const tooltipContentNode = React.useMemo(() => {
+    return <div>{counter} active filter(s)</div>;
+  }, [counter]);
```

Comment:
Why not use a string? A component prevents to get the aria label, we would need to apply it manually if we provide a component.

Reply:
> A component prevents

An element, sorry, wrong terminology.